.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
several files are useless such as this one, generated by the Ide you are using, to make sure your repo has no files related to the ide, you need to add these type of files to .gitignore file, to exclude these kind of files of being pushed